src/app/components/toolbar/toolbar.component.scss

```diff
@@ -0,0 +1,13 @@
+.toolbar{
+    background-color:#00589D;
```

Comment:
Color variables can be defined in a separate file, so you can reuse them
see: https://sass-lang.com/documentation/variables